### PR TITLE
task: validate import alias via importas

### PR DESCRIPTION
**Pull request by gssbzn**
Enable `importas` to force consistent import alias

### CODE REVIEW

**Review by jeroenvervaeke — DISMISSED** [review]

File: .golangci.yml [hunk 1]
```diff
@@ -59,8 +94,6 @@ linters-settings:
     sections:
       - standard
       - default
-    section-separators:
-      - newLine
```
Comment (gssbzn) [inline .golangci.yml:63]: invalid config

File: .golangci.yml [hunk 2]
```diff
@@ -6,10 +6,45 @@ linters-settings:
     min-occurrences: 2
   gocyclo:
     min-complexity: 15
-  govet:
-    check-shadowing: true
-  maligned:
-    suggest-new: true
```
Comment (gssbzn) [inline .golangci.yml:12]: invalid configs

File: .golangci.yml [hunk 3]
```diff
@@ -174,8 +207,6 @@ run:
     - e2e
     - unit
     - integration
-  skip-dirs:
-    - internal/mocks
```
Comment (gssbzn) [inline .golangci.yml:178]: invalid config

File: internal/kubernetes/operator/datafederation/datafederation.go [hunk 4]
```diff
@@ -24,7 +24,7 @@ import (
 	akov2common "github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/api/v1/common"
 	akov2status "github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/api/v1/status"
 	"go.mongodb.org/atlas-sdk/v20231115008/admin"
-	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	apisv1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (s-urbaniak) [inline internal/kubernetes/operator/datafederation/datafederation.go:27]: Note: this is not idiomatic in kube upstream, `v1` is.

**Comment by s-urbaniak** [comment]
one comment about the `meta/v1` import that refers to quite a bunch of changes.

File: internal/kubernetes/operator/datafederation/datafederation.go (hunk 4 above)
Comment (gssbzn) [reply internal/kubernetes/operator/datafederation/datafederation.go]: Updated

**Comment by gssbzn** [comment]
@s-urbaniak RFAL

**Review by s-urbaniak — APPROVED** [review]

**Pull request merged**